web/src/core/server/models/link.ts

```diff
@@ -0,0 +1,33 @@
+import { model, models, Schema } from "mongoose";
+
+const linkSchema = new Schema({
+  userId: {
+    type: String,
+    require: true
+  },
+  originalLink: {
+    type: String,
+    require: true
+  },
+  shortLink: {
+    type: String,
+    require: true,
+    unique: true
+  },
+  clicks: {
+    type: Number,
+    default: 0
+  },
+  Status: {
```

Comment:
This should not start with a capital letter:

```suggestion
  status: {
```

Also, what's it for an why is it a boolean?


I expected to see these properties as well

```ts
  name: {
      type: String,
     default: '',
  },
  password: {
      types: String,
  }
```